app/src/main/java/com/khanhhuy/movie_db/model/Category.java

```diff
@@ -0,0 +1,22 @@
+package com.khanhhuy.movie_db.model;
+
+/**
+ * Created by yeu_thuong on 11/5/2017.
+ */
+
+public class Category {
+    private String mTitle;
+    private static final String MORE = "More";
```

Comment:
2.8 Class member ordering

This is not the only correct solution, but it is recommended to use the following order :

Constants
Fields
Constructors
Override methods and callbacks (public or private)
Public methods
Private methods
Inner classes or interfaces